@xen-orchestra/rest-api/src/rest-api/rest-api.mts

```diff
@@ -21,6 +21,10 @@ export class RestApi {
     return this.#xoApp.getObjectsByType(type, opts)
   }
 
+  getXapiVmStats(...args: Parameters<XoApp['getXapiVmStats']>) {
+    return this.#xoApp.getXapiVmStats(...args)
+  }
+
```

Comment:
I think it should stay in vm controller if you do not want to have a massif rest-api.mts class doiing all the work 
maybe you can expose a getter of xoapp ?